src/components/input.js

```diff
@@ -50,7 +50,7 @@ export const withDebouncedChange = WrappedComponent => {
     const [internalValue, setInternalValue] = useState()
     const getInternalValue = Utils.useGetter(internalValue)
     const getOnChange = Utils.useGetter(onChange)
-    const updateParent = Utils.useInstance(() => _.debounce(250, () => {
+    const updateParent = Utils.useInstance(() => _.debounce(props.debounce || 250, () => {
```

Comment:
Should we destructure debounce from props? is it used somewhere further in?

Reply:
We are passing props in to other methods, so we should leave it in there